requirements.txt

```diff
@@ -1,12 +1,12 @@
-argcomplete==1.11.1
-colorama==0.4.3
-flake8==3.7.9
-jmespath==0.9.5
-mock==4.0.1
-pylint==2.5.3
-Pygments==2.5.2
-PyYAML==5.3.1
-six==1.14.0
-tabulate==0.8.6
+argcomplete==1.12.2
+colorama==0.4.4
+flake8==3.8.4
+jmespath==0.10.0
+mock==4.0.3
+Pygments==2.7.4
+pylint==2.6.0
+pytest==6.2.2
+PyYAML==5.4.1
```

Comment:
This `requirements.txt` is actually only used by `tox`. When knack is used by Azure CLI, only `setup.py` is honored.

Now sure if this will trigger issues like https://github.com/Azure/azure-cli/issues/16650, so bump them anyway.

Reply:
PyYAML is unpinned in https://github.com/microsoft/knack/pull/241.